main.py

```diff
@@ -38,6 +43,15 @@ def get_timelapse_filename(output_dir):
 
 
 def main():
+
+    if sys.stdout.isatty():
+        handler = logging.StreamHandler(sys.stdout)
+        logger.setLevel(logging.DEBUG)
+    else:
+        handler = logging.handlers.SysLogHandler(address='/dev/log')
+        logger.setLevel(logging.INFO)
+    logger.addHandler(handler)
```

Comment:
I'm unfamiliar with `logging` – I assume this handler behavior propagates to the rest of the process?

Reply:
Yes, it will persist for the the rest of the process.